conf/battle/monster.conf

```diff
@@ -297,3 +297,7 @@ achievement_mob_share: no
 // Should slaves teleport back to their master if they get too far during chase? (Note 1)
 // Default (Official): no
 slave_stick_with_master: no
+
+// Absolute minimum respawn time in milliseconds of a monster.
```

Comment:
Maybe leave a small comment that it is also used to delay spawning of guardians when the guild is not yet loaded to retry every x seconds?